src/Git/CommitFile.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\AutomaticReleases\Git;
+
+interface CommitFile
+{
+    public function __invoke(
+        string $repositoryDirectory,
+        string $filename,
+        string $commitMessage,
+        string $author
```

Comment:
This shouldn't be needed, in theory, given that the repository is pre-configured with an author email and name?

Reply:
Note also that git will honor any environment variables regarding current author/email

Reply:
I was unclear if this were the case, but seeing how the `CreateTagViaConsole` class is written, it must be. I'll revert that in my next revisions.